projects/plugins/boost/changelog/update-required-wp-version

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+Support: Increased minumum required PHP version to 6.5
```

Comment:
PHP 6 returns!

Reply:
Hehe, yeah. We fixed that in the release 😅